cmake/android/android_gradle_projects.cmake

```diff
@@ -82,6 +82,9 @@ foreach(fname ${GRADLE_WRAPPER_FILES})
   install(FILES "${OpenCV_SOURCE_DIR}/platforms/android/gradle-wrapper/${fname}" DESTINATION "${ANDROID_INSTALL_SAMPLES_DIR}/${__dir}" COMPONENT samples ${__permissions})
 endforeach()
 
+# set build.gradle namespace 
+set(OPENCV_NAMESPACE "org.opencv" CACHE STRING "OpenCV Namespace string")
```

Comment:
Need to use version check to avoid breaking of existed build configurations:

```
if(... GRADLE VERSION CHECK ...)
    ocv_update(OPENCV_NAMESPACE_DECLARATION "namespace 'org.opencv'")
else()
    ocv_update(OPENCV_NAMESPACE_DECLARATION "")
endif()
```

and below:

```.diff
 android {
+    @OPENCV_NAMESPACE_DECLARATION@
```

Reply:
I made those changes accordingly so please check it out.